src/test/kotlin/com/zepben/auth/server/AuthConfigRouteTest.kt

```diff
@@ -28,17 +38,17 @@ class AuthConfigRouteTest {
             RouteVersionUtils.forVersion(
                 AvailableRoute.values(),
                 2
-            ) { routeFactory(it, "test-auth-type", "test-issuer", "test-audience") }
+            ) { routeFactory(it, "test-aud", "test-domain", "test-alg") }
         )
         port = server!!.listen()
     }
 
     @Test
     fun testHandle() {
         val expectedResponse: String = JsonObject().apply {
-            put("authType", "test-auth-type")
-            put("issuer", "test-issuer")
-            put("audience", "test-audience")
+            put("aud", "test-aud")
+            put("dom", "test-domain")
+            put("alg", "test-alg")
```

Comment:
Looks like the expected keys for the route response have changed. This needs to be added to the breaking changes section of the change log if this is the case.

Reply:
This was a diff I overlooked when I copied over code from the now defunct `zepben.auth` package to this one. I'll change it back.